daemon/src/taker_cfd.rs

```diff
@@ -139,25 +141,18 @@ impl<O, W> Actor<O, W> {
 
         let proposal_closing_price = market_closing_price(bid, ask, Role::Taker, cfd.position());
 
-        let disconnected = self
-            .collab_settlement_actors
-            .get_disconnected(order_id)
-            .with_context(|| format!("Settlement for order {order_id} is already in progress"))?;
-
         tracing::debug!(%order_id, %proposal_closing_price, %bid, %ask, %quote_timestamp, "Proposing settlement of contract");
 
-        let addr = collab_settlement_taker::Actor::new(
-            order_id,
-            proposal_closing_price,
-            self.n_payouts,
-            self.conn_actor.clone(),
-            self.process_manager_actor.clone(),
-            self.db.clone(),
-        )
-        .create(None)
-        .spawn(&mut self.tasks);
-
-        disconnected.insert(addr);
+        // Sending msg sync to check for invariant
```

Comment:
🙃 I don't think "sync" is the correct term, because this is `async` code. `xtra`'s documentation says that we are sending a message to the actor and "asynchronously wait[ing] for a response".

Reply:
that is correct. I've amended the comment :)